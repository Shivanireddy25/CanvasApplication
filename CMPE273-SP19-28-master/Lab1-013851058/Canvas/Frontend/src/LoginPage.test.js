import React from 'react';
import { shallow } from 'enzyme';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
import MyComponent from '../src/components/Login/LoginPage';
describe('MyComponent', () => {
  it('Rendering Login Page', () => {
    const component = shallow(<MyComponent debug/>);
    expect(component).toMatchSnapshot();
  });
});
