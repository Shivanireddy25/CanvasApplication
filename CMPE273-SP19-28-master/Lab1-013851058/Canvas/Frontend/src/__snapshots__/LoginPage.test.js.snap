// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent Rendering Login Page 1`] = `
<div
  className="Fullpage"
>
  <div
    align="center"
    className="LoginPage"
  >
    <div
      className="Login"
    >
      <div>
        <div>
          <h3
            className="loginheader"
          >
            Connecting to 
            <span
              className="titlespan"
            >
              SJSU
            </span>
          </h3>
          <h6
            className="loginheader loginheadersub"
          >
            Sign in with your SJSU ID
          </h6>
        </div>
        <div
          className="logodiv"
        >
          <img
            alt="Logo"
            className="loginlogo"
            src="https://d92mrp7hetgfk.cloudfront.net/images/sites/misc/san_jose_state_u-1/standard.png?1548463655"
          />
        </div>
      </div>
      <form
        align="center"
      >
        <ForwardRef(Bootstrap(FormGroup))
          controlId="name"
        >
          <ForwardRef(ContextTransform)
            onChange={[Function]}
            placeholder="Name"
            type="text"
            value=""
          />
        </ForwardRef(Bootstrap(FormGroup))>
        <ForwardRef(Bootstrap(FormGroup))
          controlId="password"
        >
          <ForwardRef(ContextTransform)
            onChange={[Function]}
            placeholder="Password"
            type="password"
            value=""
          />
        </ForwardRef(Bootstrap(FormGroup))>
        <ForwardRef(Bootstrap(Button))
          block={true}
          onClick={[Function]}
          type="button"
        >
          Sign In As Student
        </ForwardRef(Bootstrap(Button))>
        <ForwardRef(Bootstrap(Button))
          block={true}
          onClick={[Function]}
          type="button"
        >
          Sign In As Faculty
        </ForwardRef(Bootstrap(Button))>
        Not a Member?  
        <Link
          replace={false}
          to="/signup"
        >
          Signup
        </Link>
        <p>
            
        </p>
      </form>
    </div>
  </div>
</div>
`;
